src/notation/internal/notation.h

```diff
@@ -50,6 +50,8 @@ class Notation : virtual public INotation, public IGetScore, public async::Async
     Meta metaInfo() const override;
     void setMetaInfo(const Meta& meta) override;
 
+    instruments::ScoreOrder scoreOrder() const override;
```

Comment:
Seems that a better place for this method is INotationPartsPtr. What do you think?

Reply:
The reason to put `scoreOrder()` in `notation.h` instead of `inotationparts.h` was `inotationparts.h` seems have have "setter" methods only, no "retrieval" methods. I didn't want to unnecessary mess up the interface.